TEMPLATE.md

```diff
@@ -0,0 +1,91 @@
+---
+RFC: <PR number>
+Status: <Proposed>
+Version: <version number>
```

Comment:
Version? How do we assign version numbers?

If the file has been updated by multiple PRs, I think we can simply mention those PRs too, so readers can find the discussion on those changes.

The README already mentions this: "For larger changes, the PR making the change is mentioned too and can be referred to by their respective pull-request numbers."

Reply:
So, I don't think we should add version. Instead the RFC field can contain stuff like `RFC: #xxx, updated by #yyy` or possibly we add a History or Updated-by field, but only for the RFCs that have updates.

Reply:
I've seen version in few of the tooling system I've used for document/code. We do have git history here so might serve a hint for someone to look at the past. 

@PingXie Do you have anything to add here ?

Reply:
People might not now what to write in a Version field. 1 or 2 or 1.1 or a date?

The PEP process has a *Created* date and a *History* with dates of changes. That's an idea that doesn't need much further documentation. Here's an example: https://peps.python.org/pep-0001/

Reply:
>People might not now what to write in a Version field. 1 or 2 or 1.1 or a date?

My main motivation is to understand why the two RFCs have different formats and I know we will change the formats at some point. 

Here is how this RFC could be used in the future:

I am about to write an RFC for feature X. I make a copy of the latest version of  this RFC, which already comes with a version without any input from me. I then fill in the sections that are applicable. done.

Not sure at which step will the author need to worry/reason about this version field?

Reply:
btw, the exact version scheme is not my concern. We can do semver or just a date like 20240819. As long as we stick to the same version scheme, I am all good.

Reply:
Ah, I see where the confusion is coming from :). Sorry, I meant to say the "meta" version - the version of this RFC template not the real RFC's version. The author can maintain a revision history for their additions/removals/modifications. I think there are a plenty of good systems out there for maintaining revision history. A simple one like "<date>: <change>" would work for me.

Reply:
I like the PEP created date / history concept. We can apply it in the future to an actual RFC. @PingXie Do we want this meta version thing ? I'm not sure anymore.

Reply:
I like the PEP format too. I can still see a use of the meta version but not a hard blocker for me.